diff --git a/platform/CMSIS/RTOS2/Source/os_systick.c b/platform/CMSIS/RTOS2/Source/os_systick.c
index 3cce53c52..d38dc1f21 100644
--- a/platform/CMSIS/RTOS2/Source/os_systick.c
+++ b/platform/CMSIS/RTOS2/Source/os_systick.c
@@ -26,7 +26,8 @@
 
 //lint -emacro((923,9078),SCB,SysTick) "cast from unsigned long to pointer"
 #include "RTE_Components.h"
-#include CMSIS_device_header
+// #include CMSIS_device_header
+#include "em_device.h"
 
 #ifdef  SysTick
 
